packages/typespec-test/test/openai_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -258,6 +262,7 @@ export async function _getImageOperationStatusDeserialize(
   result:
     | GetImageOperationStatus200Response
     | GetImageOperationStatusDefaultResponse
+    | GetImageOperationStatusLogicalResponse
```

Comment:
I think this is the correct change, as for LRO, we have a LogicalResponse in the RLC layer for type inference.